bee-message/tests/treasury_transaction_payload.rs

```diff
@@ -25,49 +16,19 @@ fn kind() {
 
 #[test]
 fn new_valid() {
-    let input = Input::from(TreasuryInput::from_str(MESSAGE_ID).unwrap());
-    let output = Output::from(TreasuryOutput::new(1_000).unwrap());
-    let transaction = TreasuryTransactionPayload::new(input.clone(), output.clone()).unwrap();
+    let input = TreasuryInput::from_str(MESSAGE_ID).unwrap();
+    let output = TreasuryOutput::new(1_000).unwrap();
+    let transaction = TreasuryTransactionPayload::new(input, output.clone()).unwrap();
```

Comment:
Can the `clone` be removed here, too?
```suggestion
    let transaction = TreasuryTransactionPayload::new(input, output).unwrap();
```

Reply:
Yes, we use it L24